apps/pie-docs/src/components/data-visualisation/data-visulisation.json

```diff
@@ -1,4 +0,0 @@
-{
-  "metaTitlePrefix": "Data Visulisation — ",
-  "permalink": "false"
```

Comment:
From what I can see, this `"false"` wasn't working before as the pages were being displayed, so I've removed it. There's enough of a diff that git thinks it's a different file to the one 2 files above ^^